templates/shit-o-lator.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Shit-o-lator v.4.1!!! ;)</title>
+    <link rel="stylesheet" type="text/css" href="/static/css/style.css">
```

Comment:
As you can see, another way is using "app = Flask(__name__, static_url_path='/static')" and then in the templates you can use usual URLs! ;-)